elections/storage/insert_tweet.py

```diff
@@ -0,0 +1,16 @@
+import pymysql
+import tweepy
+from open_connection import Connection
+
+def insert(buffer):
```

Comment:
Create a class "DatabaseConnector" or something like that instead of just a function, you need to divide functionality